playground/style.css

```diff
@@ -1451,4 +1452,37 @@
   font-weight: bold;
   background-color: #f5f7f7;
   border: solid 1px #bdc1c5;
+}
+
+.customTable-expandedContent {
+  margin: 12px;
+  background-color: #ffffff;
+}
+
+.customTable-expandedItems {
+  font-family: Muli , sans-serif;
+  font-size: 12px;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.33;
+  letter-spacing: 0.5px;
+  color: #003b5c;
+}
+
+.customTable-itemHeading {
+  font-weight: bold;
+  padding: 8px;
+}
+
+#root_procedureCodes .react-bs-table .table-bordered>thead>tr:first-child>td, .react-bs-table .table-bordered>thead>tr:first-child>th {
+  background-color: #f5f7f7;
+  border: solid 1px #dadada !important;
```

Comment:
Are these !important necessary?

Reply:
yes, There are most of the places having border css. To fix that we need to put important